docs/doc/13-monitor/prometheus-and-granafa.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Prometheus & Grafana
+---
+
+[Prometheus](https://prometheus.io/) is a time series database that features a flexible query language and a multi-dimensional data model.
+
+[Grafana](https://grafana.com/grafana) is an open-source tool used for analyzing and visualizing metrics.
```

Comment:
Alternatively, you can use Prometheus to capture and store the metrics data from Databend. Then, you can visualize the captured time series data on a dashboard with Grafana. [Prometheus](https://prometheus.io/) is an open-source monitoring system with a dimensional data model, flexible query language, efficient time series database and modern alerting approach. [Grafana](https://grafana.com/grafana) is an open-source tool used for analyzing and visualizing metrics.